firefox-ios/Client/Frontend/Library/Bookmarks/Utitlity/FolderHierarchyFetcher.swift

```diff
@@ -20,6 +20,8 @@ struct Folder: Equatable {
     let guid: String
     let indentation: Int
 
+    static let dummyFolderGuid = "DUMMY"
```

Comment:
Could we rename to something else, like "DesktopFolderHeaderPlaceholderGuid"? Kind of long, maybe you have a better idea, but otherwise you kind of need to dig in the code to understand what it's used for